SIDFactoryII/source/runtime/editor/components/component_track.cpp

```diff
@@ -130,10 +132,10 @@ namespace Editor
 		UpdateMaxEventPos();
 
 		m_FocusRow = ComponentTrackUtils::CalculateFocusRow(m_Dimensions.m_Height >> 1, m_Dimensions.m_Height);
-	
+
```

Comment:
Removed all the trailing whitespace, sorry for the mess

Reply:
I think the getfirstfree and getfirstunused sequence functions do not belong in the component base. I don't quite know why I put the firstfree one there in the first place.